src/helpers/binaries.js

```diff
@@ -0,0 +1,53 @@
+const utils = require('../utils');
+
+module.exports = {
```

Comment:
Use `esm` ?

Reply:
I thought about it. Commonjs is fine as this is a forever node only package.

Reply:
How about export individual function instead of a default exports ?

Also, I am still inclined to use `esm` since Node.js support `esm` too.
We may can export various output if we use `esm`. In ts, you can export `commonjs` and `esm` if the source is `esm` style.